openapi.yaml

```diff
@@ -434,10 +464,17 @@ paths:
       summary: health
       tags:
       - UploadControllerService
-  /uploads:
+  /uploads/storages/{storage_alias}:
```

Comment:
The storage alias should rather go into the body.

Reply:
I don't know why, but I forgot about that possibility, as that directly resolves a large part of my problem...
Thanks and changed